aci/resource_aci_fvaepg.go

```diff
@@ -173,11 +173,51 @@ func resourceAciApplicationEPG() *schema.Resource {
 				Optional: true,
 				Set:      schema.HashString,
 			},
-			"relation_fv_rs_node_att": &schema.Schema{
+			"relation_fv_rs_node_att": &schema.Schema{ 
```

Comment:
Should we also change this in the data-source?

Reply:
I believe Lionel said to not change this in the data-source for now.